src/downshift.js

```diff
@@ -247,7 +247,24 @@ class Downshift extends Component {
     if (this.getState().isOpen) {
       this.changeHighlightedIndex(amount, otherStateToSet)
     } else {
-      this.setHighlightedIndex(undefined, {isOpen: true, ...otherStateToSet})
+      // if menu was closed, will open it.
```

Comment:
In this case we know it's closed because this is the else case of an isOpen check. We can remove the comment.